internal/api/v1/models/models.go

```diff
@@ -1,7 +1,6 @@
-package models
-
-// The structures in this package encapsulate the requested and
+// Package models contains structures to encapsulate the requested and
```

Comment:
IIRC you have to start with the package name for the go docs to work ?
Also:
```
- models contains structures to encapsulate the requested and 
+ models contains structures encapsulating the request and
```

Reply:
It's recommended to start with 'Package' https://golang.org/doc/effective_go#commentary

Reply:
Ok